pretrain/validation.py

```diff
@@ -144,7 +144,11 @@ def check_for_reasonable_output(
 
 
 def compute_losses(
-    model, batches: typing.List[torch.Tensor], device: str, pad_token_id: int
+        model,
+        batches: typing.List[torch.Tensor],
```

Comment:
As written this is getting a list of of numpy ndarrays? which you then convert into torch.tensors?

Reply:
It is a list of list after I removed unnecessary numpy stacking